### ISSUE TITLE
Problems with instructions on using own API key

### ISSUE THREAD

**Issue by scaramouche1** [gauteh/lieer#292]
The instructions on "[Using your own API key](https://github.com/gauteh/lieer#using-your-own-api-key)" seem to be outdated or incomplete.  Google's Cloud Console doesn't seem to offer a way to download a `client_secret.json` file.  Instead, it directly offers an API key, but it's unclear how to pass this to lieer.

**Comment by bcspragu** [comment]
The term "API key" is likely what's confusing people here:

![image](https://github.com/user-attachments/assets/606c14aa-06f3-4e14-9025-42450f67f563)

You want the "OAuth client ID" option here, not the "API key" option

With that, it works fine for me. Creating an OAuth2 client gives  you a `client_id` and a `client_secret`, and the download button looks like this for me:

![image](https://github.com/user-attachments/assets/d9dcd4f4-329d-4e7c-b95e-e5d02292e0b9)

**Comment by gauteh** [comment]
Good point. Would be great to update the instructions with this screenshot 😁

**Comment by bcspragu** [comment]
For sure, feel free to use them! (I can also send the PR if you'd prefer)

**Comment by gauteh** [comment]
> For sure, feel free to use them! (I can also send the PR if you'd prefer)

That would be awesome!

**Comment by bcspragu** [comment]
Sounds good, writing those up now! One question for you: for the public OAuth2 credentials, the only "Redirect URI" set is `localhost:xxxx`, right? Just adding a brief section on the extremely minor risks of using the public creds

**Comment by gauteh** [comment]
I plan to remove the public credentials, see linked issue above.
